lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -144,13 +144,14 @@ if Enum.all?(
     end
 
     def handle_notification(
-          {:track_playable, {content_type, _track_id}},
+          {:track_playable, {content_type, track_id}},
           {:hls_sink_bin, stream_id},
           _ctx,
           state
         ) do
+      origin = Map.get(state.tracks, track_id).origin
```

Comment:
As we should always have `track_id` in `state.tracks` we can use `Map.fetch!` instead of `Map.get`.

Also, let's do the same for `:cleanup` notification (line 158) and update docs for `owner` option of this element (line 51)

```suggestion
      %{origin: origin} = Map.fetch!(state.tracks, track_id)
```

Reply:
We can't do that because in `:cleanup` notification there is no `track_id`